internal/syncer/connector/airtable.go

```
@@ -0,0 +1,739 @@
+		if err != nil {
+			cancel()
+			return err
+		}
+		req.Header.Set("Authorization", "Bearer "+c.accessToken)
+		resp, err := c.httpClient.Do(req)
+		cancel()
+		if err != nil {
+			lastErr = err
+		} else {
+			body, readErr := io.ReadAll(io.LimitReader(resp.Body, airtableMaxJSONResponseSize+1))
+			resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)